Dockerfile

```diff
@@ -21,7 +21,28 @@ RUN mkdir -p /opt/fits && \
   curl -fSL -o /opt/fits-1.0.5.zip http://projects.iq.harvard.edu/files/fits/files/fits-1.0.5.zip && \
   cd /opt && unzip fits-1.0.5.zip && chmod +X fits-1.0.5/fits.sh
 
-RUN mkdir /data
+RUN apt-get update && \
+      apt-get -y install sudo
+
+ARG UID=8083
+ARG GID=8083
+
+# Create the home directory for the new app user.
+RUN mkdir -p /data
+RUN chown ${UID}:${GID} /data
+
+# Create an app user so our program doesn't run as root.
+RUN groupadd -g ${GID} app
+RUN useradd -g ${GID} -l -M -s /bin/false -u ${UID} app
+RUN echo "app:app" | chpasswd && adduser app sudo
+RUN echo "app ALL=(root) NOPASSWD:ALL" > /etc/sudoers.d/app && \
+    chmod 0440 /etc/sudoers.d/app
+
+RUN chown -R app:app /data
+
+# Set the home directory to our app user's home.
+ENV HOME=/data
```

Comment:
If we want a home directory, we should just specify that when adding the user (`useradd -d /data` or something).  Or does something not work when HOME is actually assigned via the user?